CONTRIBUTING.rst

```diff
@@ -477,6 +477,23 @@ to use the `wayback machine <https://web.archive.org/>`_ to create a snapshot of
 and link the internet archive link. The URL of the resource can change, and it creates unreachable
 links from the documentation.
 
+Using Math Formulas and Latex Formatting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When working with docstrings that contain math symbols or formulas
+use raw format (r""") to ensure proper rendering. 
+While LaTeX formatting can improve the appearance of the documentation, 
+it's best to keep it simple and readable. 
+
+An example of a math formula::
+      
+      .. math:: 
+         Ax = \lambda x
+
+Some inline math::
+    
+   These are Cheeger's Inequalities for \d-Regular graphs: 
+   \frac{d- \lambda_2}{2} \leq h(G) \leq \sqrt{2d(d- \lambda_2)}
```

Comment:
Just to check that these examples work, let's put the results of that syntax into the document.
And we need to add the dollar signs around the inline text.
```suggestion
      .. math:: 
          Ax = \lambda x

.. math:: 
    Ax = \lambda x

Some inline math::
    
    These are Cheeger's Inequalities for \d-Regular graphs: 
    $\frac{d- \lambda_2}{2} \leq h(G) \leq \sqrt{2d(d- \lambda_2)}$
   
These are Cheeger's Inequalities for \d-Regular graphs: 
$\frac{d- \lambda_2}{2} \leq h(G) \leq \sqrt{2d(d- \lambda_2)}$
```